DNN Platform/DotNetNuke.Web/Startup.cs

```diff
@@ -14,6 +17,7 @@ public class Startup : IDnnStartup
         /// <inheritdoc />
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddTransient<IFilterProvider, DnnActionFilterProvider>();

```

Comment:
Does it make sense for this to be a singleton, since it's only manually accessed once and then stored in global configuration?

Reply:
If it is only instantiated once and then stored in the global configuration I think it makes sense to keep it as a transient. Otherwise it would store that singleton instance in the Dependency Injection container. If you think the `IFilterProvider` will be needed outside the global configuration then I agree we should make it a singleton.